maintenance_account/models/account_move.py

```diff
@@ -113,13 +115,20 @@ def _set_equipment_category(self):
             self.equipment_category_id = category.id
 
     def _prepare_equipment_vals(self):
+        equipment_name = self.name
+        description = False
+        if "\n" in self.name:
+            lf_index = self.name.index("\n")
+            equipment_name = self.name[:lf_index]
+            description = html_escape(self.name[lf_index + 1 :])
```

Comment:
I would say `html_escape` is not for converting text to HTML.

Reply:
https://github.com/odoo/odoo/blob/ed890131f5bff8264e152c77ae5d3757f2dd7bfb/odoo/tools/misc.py#L1332

https://markupsafe.palletsprojects.com/en/2.1.x/

Reply:
could you please check it now?

Reply:
It's still not valid:

![imagen](https://github.com/OCA/maintenance/assets/7165771/b27e383d-e5fe-4a84-b714-3ef3dd8be52f)

But I wonder if you have put a test and it's green, is it because v16 makes the conversion automatically, or is it because the test is not being executed?

In v15, I have just checked, and it's incorrect. Look at these 2 screenshots that are self-explanatory:

![imagen](https://github.com/OCA/maintenance/assets/7165771/0b6ebaac-8450-4625-ac66-4e8826803e59)

![imagen](https://github.com/OCA/maintenance/assets/7165771/7db07a03-adc1-4007-881a-031592c1fc8c)

Reply:
Could you please check it now?
I do not know why there where some error test and was checked as green over here...
On v16 it is transform automatically, but if you add \n on text wont recognise, thats why i added a replace on field.
Im checking why user_type_id are not found...